core/hotspot/rule_manager.go

```diff
@@ -137,7 +146,13 @@ func ClearRules() error {
 	return err
 }
 
-func onRuleUpdate(rules []*Rule) (err error) {
+// ClearRulesOfResource clears resource level parameter flow rules.
```

Comment:
flow rules -> hotspot rules

Reply:
`parameter flow rules` is the history name, it ` means `hotspot rules`.